Parse/Internal/Push/Utilites/PFPushUtilities.m

```diff
@@ -21,6 +21,7 @@
 #import "PFKeychainStore.h"
 #import "PFLogging.h"
 #import "PFMacros.h"
+#import "PFAlertView.h"
```

Comment:
Either wrap this in a `#if TARGET_IOS_IPHONE` check, or change `PFAlertView`'s headers to support OS X. 

I don't have a strong preference to either, though I think that long term making `PFAlertView` cross-platform (with a compile-time warning using NS_AVAILABLE and friends, if you attempt to actually call it on OS X) makes it much simpler to maintain long-term.